templates/index.html

```diff
@@ -75,6 +75,18 @@
                         element.style.display = "none";
                     }
                 }
+            } else {
+                document.getElementById("keyboard").innerHTML = og_keyboard_html;
```

Comment:
why?

Reply:
somebody was too lazy to write `document.querySelectorAll('.key').forEach(x => x.style.display = 'inline-block');` apparently